spec/dummy/app/views/layouts/application.html.erb

```diff
@@ -1,3 +1,3 @@
-<%= inertia_headers %>
+<%= local_assigns[:inertia_ssr_head] %>
```

Comment:
Agreed 👍 

I would add it with a warning that it will be removed.

These are not headers, it's `html` for the `head`, so the name is inaccurate.

In addition, it does not make sense to call this in the controller, as it wouldn't have been set yet.